libraries/app/include/graphene/app/api.hpp

```diff
@@ -518,6 +520,31 @@ namespace graphene { namespace app {
          application& _app;
          graphene::app::database_api database_api;
    };
+
+   /**
+    * @brief The custom_operations_api class exposes access to standard custom objects parsed by the
+    * custom_operations_plugin.
+    */
+   class custom_operations_api
+   {
+      public:
+         custom_operations_api(application& app):_app(app), database_api( std::ref(*app.chain_database()),
```

Comment:
Out of curiosity - why is `std::ref` used here?

Reply:
I am not sure honestly. I see that the apis that need `database_api`(`history_api`, `orders_api`) do it this way.

Reply:
Please remove.

Reply:
removed at https://github.com/bitshares/bitshares-core/pull/1926/commits/05a2cd20464037719c5a8b256741a64aef1486e3